lua/gcompute/ui/ide/plugins/metastruct_coh.lua

```diff
@@ -87,6 +87,7 @@ function self:UpdateHookedView ()
 end
 
 function self:UpdateChatStatus ()
+	local find = string.find
```

Comment:
why

Reply:
I suppose I could just have it be string.find(code,"secret",1,true) that seems a bit more cluttered maybe? I'll change it if asked.